scripts/ci/test_plan.py

```diff
@@ -103,6 +103,9 @@ def __init__(self, modified_files, pull_request=False, platforms=[], detailed_te
         self.platforms = platforms
         self.default_run = False
         self.detailed_test_id = detailed_test_id
+        self.tag_cfg_file = os.path.join(zephyr_base, 'scripts', 'ci', 'tags.yaml')
```

Comment:
same as with ignore_path? why are we doing os.path.join here and with ignore_path it is done as a fromatted string, lets use one way to do that...